cmd/lintpack/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/go-lintpack/lintpack/internal/cmdutil"
+)
+
+func main() {
+	log.SetFlags(0)
+	cmdutil.DispatchCommand(subCommands)
+}
+
+var subCommands = []*cmdutil.SubCommand{
+	{
+		Main:  lintpackBuild,
+		Name:  "build",
+		Short: "build linter from made of lintpack-compatible packages",
+	},
+	{
+		Main:  lintpackVersion,
+		Name:  "version",
+		Short: "print lintpack version",
+	},
+}
+
+func lintpackVersion() {
+	fmt.Println("unreleased/unversioned, v0.0.0")
```

Comment:
I think such 'variables` should be extracted to a real variable.
So in the future it will be easier to inject them at compile time 😉